app/views/articles/show.html.erb

```diff
@@ -0,0 +1,8 @@
+<%= provide(:title, @article.name) %>
+
+<h1 class="page-header"><%= @article.name %></h1>
+<div class="pull-right">
+  <%= link_to 'Edit article', edit_article_path(@article.id), class: "text-danger" %> | 
+  <%= link_to 'Back to index', articles_path, class: "text-danger" %>
+</div>
+<div class="clearfix"><%= @article.text.html_safe %></div>
```

Comment:
You might want to learn about HAML. Simplifies the markup greatly. This whole file, for example, would look like this:

``` haml
= provide(:title, @article.name)
%h1.page-header= @article.name
.pull-right
  = link_to 'Edit article', edit_article_path(@article.id), class: "text-danger"
  |
  = link_to 'Back to index', articles_path, class: "text-danger"
.clearfix= @article.text.html_safe
```